lib/config-migration.js

```diff
@@ -344,24 +347,29 @@ function migrateComplianceTriggers (config) {
 
   const requirements = {
     sms: 'sms',
-    idData: 'idData',
-    idPhoto: 'idPhoto',
-    facePhoto: 'facePhoto',
-    sanctions: 'sanctions'
+    idData: 'idCardData',
+    idPhoto: 'idCardPhoto',
+    facePhoto: 'facephoto',
+    sanctions: 'sanctions',
+    suspend: 'suspend'
   }
 
   function createTrigger (
     requirement,
-    threshold
+    threshold,
+    suspensionDays = null
   ) {
-    return {
+    const triggerConfig = {
       id: uuid.v4(),
-      cashDirection: 'both',
+      direction: 'both',
+      suspensionDays: suspensionDays,
```

Comment:
Ok, I still have one nitpick here. We'll be creating suspensionDays for all triggers but that should be done only for suspend. 

```js
if (!triggerType === 'suspend) return triggerConfig

return _.assign(triggerConfig, { suspensionDays })
```